### ISSUE TITLE
[MAS2.1.1] Interactive controls present in Log section are not accessible using keyboard. 

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#747]
**Actual Result:**
Interactive controls (i.e. links) present in Log section are not accessible using keyboard.

**Expected Result:**
Interactive controls (i.e. links) present in Log section should be accessible using keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Verify that live chat tab is open.
4. Notice Log sections present in right side.
5. Navigate to 'Log' section and try to navigate through and select interactive control present in this  section using keyboard.

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}&action=view&wdAccPdf=0&wdparaid=ABCCC12)

Attachment:
[MAS2.1.1_BotFrameworkEmulator_LogSection_InteractiveElements.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2269284/MAS2.1.1_BotFrameworkEmulator_LogSection_InteractiveElements.zip)
